lib/utils/quit.ts

```diff
@@ -1,4 +1,4 @@
 export function quit(message: string | null, exitCode = 2) {
   if (message) console.log(`\n${message}\n`);
-  process.exitCode = exitCode;
```

Comment:
This was incorrect - setting the exit code ensured that when the process exited of other means, it would have the specified status, but the intent of the `quit()` function in all instances of usage is to immediately terminate the running process.